ovs/lib/update.py

```diff
@@ -17,382 +17,715 @@
+            UpdateController._log_message('Error during update: {0}'.format(ex), severity='error')
+            UpdateController._remove_lock_files([update_file, update_ongoing_file], ssh_clients)
         finally:
             filemutex.release()
 
     @staticmethod
-    def _log_message(message, client_ip=None, severity='info'):
-        if client_ip is not None:
-            message = '{0:<15}: {1}'.format(client_ip, message)
-        if severity == 'info':
-            UpdateController._logger.info(message, print_msg=True)
-        elif severity == 'warning':
-            UpdateController._logger.warning(message, print_msg=True)
-        elif severity == 'error':
-            UpdateController._logger.error(message, print_msg=True)
-
-    @staticmethod
-    def _remove_lock_files(files, ssh_clients):
-        for ssh_client in ssh_clients:
-            for file_name in files:
-                if ssh_client.file_exists(file_name):
-                    ssh_client.file_delete(file_name)
-
-    @staticmethod
-    def _change_services_state(services, ssh_clients, action):
+    def change_services_state(services, ssh_clients, action):
         """
         Stop/start services on SSH clients
         If action is start, we ignore errors and try to start other services on other nodes
         """
+        services = list(services)
         if action == 'start':
             services.reverse()  # Start services again in reverse order of stopping
         for service_name in services:
             for ssh_client in ssh_clients:
                 description = 'stopping' if action == 'stop' else 'starting' if action == 'start' else 'restarting'
                 try:
                     if ServiceManager.has_service(service_name, client=ssh_client):
-                        UpdateController._log_message('{0} service {1}'.format(description.capitalize(), service_name),
-                                                      ssh_client.ip)
                         Toolbox.change_service_state(client=ssh_client,
                                                      name=service_name,
                                                      state=action,
                                                      logger=UpdateController._logger)
-                        UpdateController._log_message('{0} service {1}'.format('Stopped' if action == 'stop' else 'Started' if action == 'start' else 'Restarted', service_name), ssh_client.ip)
                 except Exception as exc:
                     UpdateController._log_message('Something went wrong {0} service {1}: {2}'.format(description, service_name, exc), ssh_client.ip, severity='warning')
                     if action == 'stop':
                         return False
         return True
+
+    ###########
+    # HELPERS #
+    ###########
+    @staticmethod
+    def _log_message(message, client_ip=None, severity='debug'):
```

Comment:
I really don't like this wrapping everywhere. Since we tend to log the ip related to the thing we log, can't we just update our log handler to it can take an optional client, where it adds it's ip - if available - to the message?